R/plot_fns.R

```diff
@@ -6490,7 +6480,7 @@ make_volcano_plot_df <- function(x) {
     gsub(
       pattern = "_vs_",
       replacement = " vs ",
-      volcano$Comparison
+      attributes(x)$comparisons
```

Comment:
since `attributes(x)$comparisons` is just a vector of the comparison names, this just assigned a repeating vector to volcano$Comparison, which I don't think is aligned?

I'm thinking we can still do the `pvals$Comparison` reassignment before the merge(), but use an sapply over `pvals$Comparison` and replace it with whatever each element matches to in `paste0(attr(x, "comparisons"), "$")` or something....except we still need to catch annoying edge cases like one element of attr(x, "comparisons") being a substring of another....